### PR TITLE
Added asymmetric JWS key settings for edxapp

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-3629

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/devops please review

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -913,6 +913,8 @@ generic_env_config:  &edxapp_generic_env
   LOG_DIR:  "{{ COMMON_DATA_DIR }}/logs/edx"
   JWT_ISSUER: "{{ EDXAPP_LMS_ISSUER }}"
   JWT_EXPIRATION: '{{ EDXAPP_JWT_EXPIRATION }}'
+  JWT_PRIVATE_SIGNING_KEY: null
```
Comment (jibsheet) [inline playbooks/roles/edxapp/defaults/main.yml]: Do you mean null or !!null?

Comment (clintonb) [reply playbooks/roles/edxapp/defaults/main.yml]: Question I should have asked 1-2 years ago: what's the difference?

**Review by jibsheet — APPROVED** [review]

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (clintonb) [reply playbooks/roles/edxapp/defaults/main.yml]: Updated.

**Pull request merged**